packages/neuron-wallet/src/services/tx/transaction-generator.ts

```diff
@@ -331,15 +357,15 @@ export class TransactionGenerator {
 
     // change
     let finalFee: bigint = feeInt
+    const lockHashes = new Set(allInputs.map(i => i.lockHash!))
+    const keyCount: number = lockHashes.size
+    const txSize: number =
+      TransactionSize.tx(tx) +
+      (multisigConfig
+        ? TransactionSize.multiSignWitness(multisigConfig.r, multisigConfig.m, multisigConfig.n)
+        : TransactionSize.secpLockWitness() * keyCount) +
+      TransactionSize.emptyWitness() * (allInputs.length - keyCount)
```

Comment:
could you talk about the reason for moving this part out of the covering of `mode.isFeeRateMode` condition?

Reply:
I need to return the tx size to the front end.